error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ClassRulesTestInput.java

```diff
@@ -0,0 +1,25 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.common.collect.ImmutableSet;
+import java.io.IOException;
+import tech.picnic.errorprone.refaster.test.RefasterRuleCollectionTestCase;
+
+final class ClassRulesTest implements RefasterRuleCollectionTestCase {
+  boolean testClassIsInstance() throws IOException {
+    return CharSequence.class.isAssignableFrom("foo".getClass());
+  }
+
+  ImmutableSet<Boolean> testInstanceof() throws IOException {
+    Class<?> clazz = CharSequence.class;
+    return ImmutableSet.of(CharSequence.class.isInstance("foo"), clazz.isInstance("bar"));
```

Comment:
Hmm, that means we can actually drop this second test case right? What's the reason for keeping it?

Reply:
We could, but I wanted to be explicit about covering both cases. We could keep _just_ the second, if you prefer, cause that one is more generic (but that's only clear in the context of the other rules).

Reply:
AHh okay clear, let's keep it 😄.